lib/adiwg/mdtranslator/writers/sbJson/sections/sbJson_sbJson.rb

```diff
@@ -47,9 +48,15 @@ def self.build(intObj, responseObj)
                   json.summary resourceInfo[:shortAbstract]
                   json.citation Citation.build(hCitation) unless hCitation.empty?
 
-                  # do not duplicate the identifier which is the primary resource
-                  # build new array of identifiers
+                  # gather all identifiers
+                  # include the metadataIdentifier if it is not in the 'gov.sciencebase.catalog' namespace
                   aIdentifiers = []
+                  unless metadataInfo[:metadataIdentifier].empty?
+                     unless metadataInfo[:metadataIdentifier][:namespace] == 'gov.sciencebase.catalog'
+                        aIdentifiers << metadataInfo[:metadataIdentifier]
+                     end
+                  end
+                  # do not duplicate the identifier which is the primary resource
```

Comment:
This only applies to the metadataIdentifiers with the ScienceBase namespace. Need to check for that here. Otherwise, the uuid metadata identifiers will almost always be left out, since they are the default metadataIds in the mdEditor.

Reply:
Nevermind, I see the previous code block takes care of this.